stdlib/hashtbl.mli

```diff
@@ -120,7 +120,19 @@ val iter : ('a -> 'b -> unit) -> ('a, 'b) t -> unit
    in which the bindings are enumerated is reproducible between
    successive runs of the program, and even between minor versions
    of OCaml.  For randomized hash tables, the order of enumeration
-   is entirely random. *)
+   is entirely random.
+
+   The behavior is not defined if the hash table is modified
+   by [f] during the iteration.
+*)
+
+val update: ('a -> 'b -> 'b option) -> ('a, 'b) t -> unit
+(** [Hashtbl.update f tbl] applies [f] to all bindings in table [tbl]
+    and update each binding depending on the result of [f].  If [f]
+    returns [None], the binding is discared.  If it returns [Some
+    new_val], the binding is update to associare the key to [new_val].
+
+    Other comments for {!Hashtbl.update} apply as well.  *)
```

Comment:
I meant Hashtbl.iter.  This is to avoid copying most of the doc text from Hashtbl.iter.